### ISSUE TITLE
Migrating ROS message templates from Bitbucket

### ISSUE THREAD

**Issue by rohanb2018** [mit-uav/uav-msgs#1]
Hey guys,

I think it would be really helpful to migrate the ROS message templates that we had in Bitbucket (https://bitbucket.org/mrkris13/uav-msgs/src) to the Github, just so we have everything in the same place. What do you guys think?

--Rohan

**Comment by suryabhupa** [comment]
Yeah, I agree -- is there a list of messages that are on bitbucket and not on our Github?

**Comment by squilter** [comment]
If the bitbucket one is newer, do we just want to force push the whole thing?
